### ISSUE TITLE
MacBook M1: Broken sensor naming in Stats v2.8.21. Was correct in v2.6.2.

### ISSUE THREAD

**Issue by germanicianus** [exelban/stats#1472]
See attached screenshots for comparison.
<img width="300" alt="stats_v2 6 2_sensor_names_readable" src="https://github.com/exelban/stats/assets/37223176/08601aa4-4c9a-4042-8aa0-c66f7ba0dd56"><img width="300" alt="stats_v2 8 21_sensor_names_cryptic" src="https://github.com/exelban/stats/assets/37223176/84f94cde-450c-406f-900f-e192ca740482">

**Comment by exelban** [comment]
looks like you have enabled different sensors. You can enable what you want to see in the settings.

**Comment by germanicianus** [comment]
@exelban, no I don't think so. I only enabled _Show unknown sensors_ in the settings. The sensors list in above screenshot shows ALL available sensors (which is correct to get an overview). Enabling sensors adds them to the menu bar at the top. See below screenshot depicting disabled sensors being shown in the sensors list (which is correct).
<img width="747" alt="stats_v22 6 2_enabled_sensors_vs sensor_list" src="https://github.com/exelban/stats/assets/37223176/12e5c42e-c5f5-4713-8644-ba880b992313">

**Comment by exelban** [comment]
still, I will not even try to understand why it's missed. There were too many changes for almost 2 years. 
It's a git, so you can figured out why it's missed.

**Comment by germanicianus** [comment]
I figured it out. The _HID sensors_ option was a good hint. This option is not present in the old version, but in the new option, where it is enabled by default. The only thing I did in the new version was to enable _Show unknown sensors_.

I solved the problem now by disabling and re-enabling the option _HID sensors_. Afterwards I disabled the option _Show unknown sensors_ because this is not necessary for the sensors I need to show.

Therefore there seems to be a settings glitch regarding the option _HID sensors_ when updating (I did that within the program). Maybe that should be mentioned somewhere (if not already done). Somewhere in the doc I've read that reading out the HID sensors has more power consumption.

Thanks for your quick answers.
